pkg/kubelet/cm/dra/plugin/client.go

```diff
@@ -117,16 +85,12 @@ func (r *draPluginClient) NodePrepareResource(
 		"claimName", claimName,
 		"resourceHandle", resourceHandle)
 
-	if r.nodeV1ClientCreator == nil {
-		return nil, errors.New("failed to call NodePrepareResource. nodeV1ClientCreator is nil")
+	if !r.plugin.IsConnReady() {
+		if err := r.plugin.CreateGRPCConn(); err != nil {
+			return nil, err
+		}
```

Comment:
Instead of having two separate calls here, can we combine this into a single `GetOrCreateGRPCConn()` And then creete the client from the result right after it.

Reply:
done in ef1a07b143e